### PR TITLE
instructorFeedbackSessionAdd form: Give an edit links to both sentences #2305

### LINKED ISSUE #2305: instructorFeedbackSessionAdd form: Give an edit links to both sentences
Currently, the edit link is given to the 2nd sentence only. Some users might think the session visibility (first sentence) cannot be changed.
By the way, 'change' may be better than 'edit'? 'change' is the intent, 'edit' is the mechanism.

### PR DESCRIPTION
**Pull request by xpdavid**
Fixes #2305

**Changes in code:**
The HTML layouts and JS files are changed. Server's logic remains unchanged.
8aaabff4c7294eab26bdd731d4f81af78aa961ff
93a06f3f49426001130fbc124bde8577bcfc7719

**Fix test cases:**
fix InstructorFeedbackEditPageUiTest: 9c77e2bbab3d3aaa18b1eb10f753b290bdff6aaa	
fix InstructorFeedbackPageUiTest: 77dd271e1a43cb18b915b8554f34d6478778d6e6
fix AdminHomePageUiTest: 9b811308b9100beb0139a84e8099c3efe927b356
fix FeedbackConstSumOptionQuestionUiTest: 0845f38cb66aad56487c2c3a8b57090db7e2bc76
fix FeedbackConstSumRecipientQuestionUiTest: 83b8407a8951822eefe4e0bdde4a73e93dcd785c
fix FeedbackContributionQuestionUiTest: 10330f2c012d01af54bcaca04ff2cd771c7c67b8
fix FeedbackMcqQuestionUiTest: 98cd2f0d7a77a2ac49f923b654ecf44c1101dda7
fix FeedbackMsqQuestionUiTest: 06148b73d6b47bc16c8faf7b706f29ffed3eca16
fix FeedbackNumScaleQuestionUiTest: a856aceffb72e797efdba18c32ae5cefd57444b9
fix FeedbackRankQuestionUiTest: 7cecfc83f4e1f867dfa40a0f0f135ba505e8df20
fix FeedbackRubricQuestionUiTest: 021c567b9bf96f24835b88e4a86aea280ed71d38
fix InstructorFeedbackEditCopyUiTest: c0222f3cdd7b0e435fa07ebd174b2ac929ae7d9a

UI demo:
**Before:**
The [edit] link is in the second paragraph.
screen shot 2017-01-15 at 11 21 33 pm
When click it, two forms are expended.
screen shot 2017-01-15 at 11 21 43 pm

**After:**
Two [Change] links are added for paragraphs respectively.
screen shot 2017-01-15 at 11 22 14 pm
When click the one for "session visibility settings"
screen shot 2017-01-15 at 11 22 19 pm
When click the one for "session email sending options"
screen shot 2017-01-15 at 11 22 22 pm

### CODE REVIEW

File: src/test/resources/pages/instructorFeedbackAddSuccess.html [hunk 1]
```diff
@@ -986,7 +992,8 @@ <h5 class="col-sm-2">
                   <input id="noofchoicecreated--1" name="noofchoicecreated" type="hidden" value="2">
                 </div>
                 <div class="col-sm-6 col-lg-5 col-lg-offset-1 padding-right-25px">
-                  <div class="border-gray narrow-slight visible-xs margin-bottom-7px margin-top-7px"></div>
```
Comment (xpdavid) [inline src/test/resources/pages/instructorFeedbackAddSuccess.html]: A line breaker is added in every expected `html` file by "GodMod" for some reasons. 
@wkurniawan07 says should be fine.

**Comment by xpdavid** [comment]
We actually separate the uncommon settings form into two section (one [Change] link is for one session visibility uncommon settings and the other is for sending emails uncommon settings)

When instructor edit session, a `js` script will detect whether the uncommon settings are default values or not. If not, the uncommon settings will be expended. Currently, two sections will be expended although there is only one settings not in default values. The desired behaviour is only to expend section whose settings are not default values.

I prefer to create an another issue and PR for this feature.

File: src/test/resources/pages/instructorFeedbackAddSuccess.html (hunk 1 above)
Comment (wkurniawan07) [inline src/test/resources/pages/instructorFeedbackAddSuccess.html]: Indeed, it seems like someone did not use God mode when modifying these files before. Let changes like this pass.

**Comment by chowyb** [comment]
Leaving this as a comment first and deferring the decision as to whether this should be fixed to @damithc, as while I feel that the situation described below may be a problem, fixing it may involve changing the layout significantly more than necessary.

In the case where a user decides to first customize the email settings, he/she will first click on the link to change email settings, resulting in this screen:

slot01

Thereafter, the user may then decide to change visibility settings, where he/she will click on the link to change visibility settings at the bottom of the panel.

However, upon clicking this, the section to change visibility settings shows up in the middle of the panel instead:

slot02

Perhaps it might be better to have the visibility settings placed at the bottom instead in this scenario, but trying to handle this scenario may take a bit of work.

**Comment by xpdavid** [comment]
@chowyb 
It's doable for the way you indicated.
demo-2
But I actually prefer the second way.
demo2

**Comment by damithc** [comment]
My preference is this option (more intuitive?):

<details>
  <summary>to modify A, click here </summary>

Details of A 
</details>


<details>
  <summary>to modify B, click here </summary>

Details of B 
</details>

@chowyb is that what you are suggesting?

**Comment by chowyb** [comment]
@xpdavid yep the second way looks better 👍 

@damithc not exactly, my concern was with the options appearing out of order when clicked (ie. [emails, then visibility] in the first screenshot, followed by [visibility, then emails] in the second screenshot). Changing the messages could benefit the user though.

**Comment by damithc** [comment]
> my concern was with the options appearing out of order when clicked

Got it. Yes, the 2nd one is better. The order should not change upon clicking. It's not good UX when things jump around.

**Comment by xpdavid** [comment]
@damithc @chowyb Pls refer to #6382 instead.

**Pull request closed without merging**